Car_owners/tests/test_serializers.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+import pytest
+from rest_framework import serializers
+from application.models import Owner, Car
+from application.serializers import OwnerSerializer, CarSerializer
+
+
+def get_owner_data(n: int = 0) -> dict[str, str]:
+    name = "Adam"
+    surname = "Knafel"
+    phone = "123456789"
+
+    if n == 1:
```

Comment:
what is this?

Reply:
I think you want to use fixtures here, they can be parametrized if you want

Reply:
That is just a function, which gets particular owner's data (different for different tests). It is called inside parametrize decorator instead of writing several dictionaries by hand. I thought, that is a good idea, but if it is not good practise, I can write dictionaries with owner's data manually inside the @pytest.mark.parametrize.

Reply:
Yes, better to use parametrize or separate fixtures, this is a super weird construction